src/main/groovy/com/jakewharton/sdkmanager/internal/PackageResolver.groovy

```diff
@@ -279,11 +288,12 @@ class PackageResolver {
   }
 
   def findDependenciesStartingWith(String prefix) {
-    def deps = []
+    def deps = new HashSet()
     for (Configuration configuration : project.configurations) {
-      for (Dependency dependency : configuration.dependencies) {
-        if (dependency.group != null && dependency.group.startsWith(prefix)) {
-          deps.add dependency
+      ResolutionResult result = configuration.getIncoming().getResolutionResult()
```

Comment:
This line will make internet connection if necessary. Is there any way to limit configurations scope or make a batch resolution? Will this call be cached at the daemon level?